common/taskana-common/src/main/resources/sql/db2/taskana-schema-db2.sql

```diff
@@ -294,26 +294,27 @@ CREATE SEQUENCE SCHEDULED_JOB_SEQ
   CACHE 10;
 
 
+----------------------------------------------------------------------------------------------------
 -- LIST OF RECOMMENDED INDEXES
 -- This list is provided on a as-is basis. It is used for tuning of the internal performance tests.
 -- The script needs to be reviewed and adapted for each indiviual TASKANA setup.
 -- ===========================
 -- index[1],    0,013MB
    CREATE UNIQUE INDEX "DB2ADMIN"."IDX1805212017540"
-   ON "TASKANA "."CLASSIFICATION" ("ID" ASC) INCLUDE
+   ON "CLASSIFICATION" ("ID" ASC) INCLUDE
    ("CUSTOM_8", "CUSTOM_7", "CUSTOM_6", "CUSTOM_5", "CUSTOM_4",
    "CUSTOM_3", "CUSTOM_2", "CUSTOM_1", "APPLICATION_ENTRY_POINT",
    "SERVICE_LEVEL", "PRIORITY", "DESCRIPTION", "NAME",
    "CREATED", "VALID_IN_DOMAIN", "DOMAIN", "TYPE", "CATEGORY",
    "PARENT_KEY", "PARENT_ID", "KEY") ALLOW REVERSE SCANS COLLECT SAMPLED DETAILED STATISTICS;
    COMMIT WORK ;
 -- index[2],    3,646MB
-   CREATE INDEX "DB2ADMIN"."IDX1805212018000" ON "TASKANA "."TASK"
+   CREATE INDEX "DB2ADMIN"."IDX1805212018000" ON "TASK"
```

Comment:
"DB2ADMIN" is a schema name as well and should be removed as well.

Reply:
True, but wouldn't this mean that we create the indixes in our "TASKANA" schema as we set this as our current schema in the first line? Do we want to implement this behavior?

Reply:
We agreed to put them in the "TASKANA" schema.